### PR TITLE
Create database structure for reviews and categories

### PR DESCRIPTION
**Pull request by avakrishn**
Created database structure for reviews and categories

### CODE REVIEW

File: db/schema.sql [hunk 1]
```diff
@@ -219,3 +241,59 @@ CREATE TABLE notifications (
...
+	seller_id INT NOT NULL,
+	rating INT NOT NULL DEFAULT 0,
+	body TEXT NULL,
+	date_reviewed TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	PRIMARY KEY (id),
+	FOREIGN KEY (buyer_id) REFERENCES users(id),
+	FOREIGN KEY (seller_id) REFERENCES users(id)
+);
+
+CREATE TABLE sellers_reviews_buyers(
```
Comment (Yanananan) [inline db/schema.sql]: the table above starts with buyer (singular), this table starts with sellers (plural).  intentional?  can discuss later

Comment (avakrishn) [reply db/schema.sql]: Good catch, I have updated the table names to all be plural.

File: db/schema.sql [hunk 2]
```diff
@@ -219,3 +273,66 @@ CREATE TABLE notifications (
...
+	deal_id INT NOT NULL,
+	title VARCHAR (255) NOT NULL,
+	body TEXT NULL,
+  	date_reviewed TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	rating INT NOT NULL DEFAULT 0,
+	verified_purchase BOOLEAN NOT NULL DEFAULT FALSE,
+	likes INT DEFAULT 0,
+	dislikes INT DEFAULT 0,
+	helpful_review INT DEFAULT 0,
+	display_review BOOLEAN NOT NULL DEFAULT TRUE,
```
Comment (lawrenceslng) [inline db/schema.sql]: display_review should be false as default, update to true only if title + body pass language restriction

File: db/schema.sql [hunk 3]
```diff
@@ -219,3 +278,78 @@ CREATE TABLE notifications (
...
+);
+
+CREATE TABLE buyers_reviews_sellers(
+	id INT NOT NULL AUTO_INCREMENT,
+	buyer_id INT NOT NULL,
+	seller_id INT NOT NULL,
+	deal_id INT NOT NULL,
+	rating INT NOT NULL DEFAULT 0,
+	body TEXT NULL,
+	date_reviewed TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
```
Comment (lawrenceslng) [inline db/schema.sql]: add display_review BOOLEAN NOT NULL DEFAULT FALSE as in buyers_reviews_deals TABLE

**Pull request merged**